### ISSUE TITLE
movewindow with vfr on

### ISSUE THREAD

**Issue by mihneabuz** [hyprwm/Hyprland#1100]
Steps to reproduce
`no_vfr = false`

Expected outcome
Using `movewindow` should smoothly move the window using the animation settings  

Noted outcome
 - sometimes `movewindow` just moves the window after a short delay with no animation
 - other times, the animation is running at a much lower frame rate

Tested on Nvidia gpu and amd laptop apu

**Comment by vaxerski** [comment]
uhh you mean igpu? tf is apu

**Comment by mihneabuz** [comment]
yeah igpu

**Comment by vaxerski** [comment]
that is bery odd, I run my devices on no vfr false and no issues to be seen.

**Comment by mihneabuz** [comment]
Perhaps it's not just vfr but a combination of settings? Will do more testing

**Comment by DashieTM** [comment]
Could you post your config, and can you check whether or not other things like dragging/resizing are affected ?

**Comment by mihneabuz** [comment]
Here is a simple config that produces the issue

```
monitor=DP-1,3840x2160@120,auto,1.5

decoration {
    blur = yes
    blur_size = 4
    blur_passes = 1
    blur_new_optimizations = on

    drop_shadow = false

    dim_inactive = true
    dim_strength = 0.05
}

animations {
    enabled = yes
    bezier = myBezier, 0.2, 0.9, 0.2, 1.0
    animation = windows, 1, 2, myBezier
}

misc {
  no_vfr = false
}

$mainMod = SUPER

# Move windows with mainMod - SHIFT + arrow keys
bind = $mainMod SHIFT, h, movewindow, l
bind = $mainMod SHIFT, l, movewindow, r
bind = $mainMod SHIFT, k, movewindow, u
bind = $mainMod SHIFT, j, movewindow, d
```

Removing the blur and the dim_inactive also improve the framerate but not fix it completely

Dragging and resizing are not affected, only `movewindow` from what I can tell

**Comment by DashieTM** [comment]
drop_shadows is the issue.

**Comment by mihneabuz** [comment]
Confirmed, setting `drop_shadow=true` fixes it on both my devices

**Comment by vaxerski** [comment]
fixed in 75b7e661e798b9a32ccb00e0c7e4362c540ff4a9
